### PR TITLE
virtcontainers: fix codes misunderstanding in virtcontainers

### PR DESCRIPTION
**Pull request by jshachm**
Still there are some codes left which
will cause some misunderstanding

Change `p` in short of `pod` into `s` or `sandbox`

Fixes: #325

Signed-off-by: Haomin <caihaomin@huawei.com>

### CODE REVIEW

**Comment by katabuilder** [comment]
PSS Measurement:
Qemu: 144180 KB
Proxy: 4758 KB
Shim: 8812 KB

Memory inside container:
Total Memory: 2045972 KB
Free Memory:  2011092 KB

File: virtcontainers/api.go [hunk 1]
```diff
@@ -60,7 +60,7 @@ func createSandboxFromConfig(sandboxConfig SandboxConfig) (*Sandbox, error) {
 		return nil, err
 	}
 
-	// The sandbox is completely created now, we can store it.
+	// The s is completely created now, we can store it.
```
[the remark this answers is not in the document]
Comment (jshachm) [reply virtcontainers/api.go]: sry for that using `refactor` in `Goland IDE` and it did silly thing...

**Comment by katabuilder** [comment]
PSS Measurement:
Qemu: 146008 KB
Proxy: 6712 KB
Shim: 11074 KB

Memory inside container:
Total Memory: 2045972 KB
Free Memory:  2011092 KB

**Comment by caoruidong** [comment]
LGTM

File: virtcontainers/api.go [hunk 2]
```diff
@@ -129,27 +129,27 @@ func StartSandbox(sandboxID string) (VCSandbox, error) {
 	defer unlockSandbox(lockFile)
 
 	// Fetch the sandbox from storage and create it.
-	p, err := fetchSandbox(sandboxID)
+	sandbox, err := fetchSandbox(sandboxID)
```
Comment (bergwolf) [inline virtcontainers/api.go]: Please use `s` to be consistent with other places.

Comment (jshachm) [reply virtcontainers/api.go]: There are still some `sandbox` not `s` in functions like `StatusSandbox`.May be I should change them to `s`

@bergwolf  What do you think?

Comment (bergwolf) [reply virtcontainers/api.go]: @jshachm Yes, please. Thanks!

**Comment by katabuilder** [comment]
PSS Measurement:
Qemu: 142178 KB
Proxy: 6781 KB
Shim: 9039 KB

Memory inside container:
Total Memory: 2045972 KB
Free Memory:  2011340 KB

**Comment by caoruidong** [comment]
Since this pr only includes rename, merge it regardless of codecov.

**Pull request merged**